Content.Client/Lobby/LobbyUIController.cs

```diff
@@ -161,6 +162,10 @@ public void OnStateExited(LobbyState state)
     public void ReloadCharacterSetup()
     {
         RefreshLobbyPreview();
+
+        if (_profileEditor is { Visible: true })
+            return;
+
```

Comment:
Isn't this going to prevent legit scenarios from ever triggering, e.g. waiting on data from server / hot reloads because now nothing can get reloaded.

Reply:
I didn't think about it, I will fix it.

Reply:
> Isn't this going to prevent legit scenarios from ever triggering, e.g. waiting on data from server / hot reloads because now nothing can get reloaded.

I decided to just remove it, since it is not really needed.